spec/index.html

```diff
@@ -962,8 +962,8 @@ <h3>Abbreviating IRIs: <code>rdf:ID</code> and <code>xml:base</code></h3>
       ../../rdf11/rdf-xml/index.html#rdfms-difference-between-ID-and-about-error1
       ">
       The <code>rdf:ID</code> attribute on a <a>node element</a>
-    (not <a>property element</a>,
-    that has another meaning) can be used instead of
+    (not on <a>property element</a>;
```

Comment:
Sorry, I should have included this earlier.
```suggestion
    (not on a <a>property element</a>;
```